### ISSUE TITLE
Keystrokes applied on "My Library" or "Get new files" page are coming to effect on the Home page/ Help page of Kiwix Application once user navigates there

### ISSUE THREAD

**Issue by ghost** [bugbash/selenium-kiwix#25]
OS - Win 7
1. Open Kiwix installed from desktop icon. (Help Page appears)
2. Click on the "File --> Browse Library" from Header. (Library page appears)
3. Now, press keyboard keys (Ctrl + A) or (Ctrl + F) or (Ctrl + (PLUS)) when on this "My Library" or "Get new files" page. (These keystrokes won't work here)
4. Click on 'Hide Library' icon on header. (Help Page appears)

Expected - Applied keystroke should appear on the page where applied

![4_a2_installed_library or helppg_applied keystrokes on library pg gets applied on home pg](https://cloud.githubusercontent.com/assets/8665614/4162531/650eb80a-34dc-11e4-81ad-8fb3f8656aa1.png)

**Comment by kelson42** [comment]
Yes, I confirm the problem. It's no possible to provide these features for the library, but we can disable them.
